app/database/db.go

```diff
@@ -314,6 +314,17 @@ func (conn *DB) Count(dest interface{}) *DB {
 	return newDB(conn.db.Count(dest))
 }
 
+// Pluck used to query single column from a model as a map
+//     var ages []int64
+//     db.Find(&users).Pluck("age", &ages)
+func (conn *DB) Pluck(column string, value interface{}) *DB {
+	reflectValue := reflect.ValueOf(value).Elem()
+	if reflectValue.Len() > 0 {
+		reflectValue.Set(reflect.MakeSlice(reflectValue.Type(), 0, reflectValue.Cap()))
```

Comment:
What (and why) are you doing here? (a comment in the code may be interesting as well)

Reply:
fixed